docs/upgrade-guides/v0.10-->v1.0.md

```diff
@@ -0,0 +1,396 @@
+
+
+Changes:
+
+- Previously this endpoint returned transactions from BigchainDB's `BACKLOG`
+and from blocks marked as `UNDECIDED`. With version 1.0, this endpoint will
+only return transactions included in blocks marked as `VALID`.
+
+
+### `POST /api/v1/transactions`
+
+Documentation:
+
+- [Old](https://docs.bigchaindb.com/projects/server/en/v0.10.2/http-client-server-api.html#post--api-v1-transactions)
+- [New](https://docs.bigchaindb.com/projects/server/en/v1.0.0/http-client-server-api.html#post--api-v1-transactions)
+
+
+Changes:
+
+- A `Location` HTTP header was included in the endpoint's response to allow
+  users to check the transaction's status more easily via the
+  `/statuses?transaction_id` endpoint
+
+
+### `GET /api/v1/outputs`
+
+Documentation:
+
+- [Old](https://docs.bigchaindb.com/projects/server/en/v0.10.2/http-client-server-api.html#get--api-v1-outputs?public_key=public_key)
+- [New](https://docs.bigchaindb.com/projects/server/en/v1.0.0/http-client-server-api.html#get--api-v1-outputs?public_key=public_key)
+
+
+Changes:
+
+- Reversed the behavior of the `unspent` query parameter to `spent`, implying
+  the following behavior:
+    - If `?spent=true`, the response is an array of all spent outputs
+      associated with a given public key
+    - If `?spent=false`, response is an array of all NOT YET spent (or
+      "unspent" outputs associated with a given public key
+    - If no ``spent=` filter is present in the request, the response is an
+      array of all outputs associated with a given public key (spent and
+      unspent)
+
+Previously the response included a list of relative URLs pointed to
+transations' outputs:
+
+```json
+// Old
+[
+    "../transactions/2d431073e1477f3073a4693ac7ff9be5634751de1b8abaa1f4e19548ef0b4b0e/outputs/0"
+]
+
+// New
+[
+    {
+        "output": 0,
+        "transaction_id": "2d431073e1477f3073a4693ac7ff9be5634751de1b8abaa1f4e19548ef0b4b0e"
+    }
+]
+```
+
+In the future, we're planning to [upgrade this endpoint
+further](https://github.com/bigchaindb/bigchaindb/blob/99499b1f8783719a082813912ac9a0d363ae278f/bdb-ip.md#6-a-new-outputs-endpoint)
+to meet the requirements of [our
+users](https://github.com/bigchaindb/bigchaindb/issues/1227#issuecomment-307297473).
+
+
+### `GET /api/v1/statuses?tx_id`
+
+Documentation:
+
+- [Old](https://docs.bigchaindb.com/projects/server/en/v0.10.2/http-client-server-api.html#get--api-v1-statuses?tx_id=tx_id)
+- [New](https://docs.bigchaindb.com/projects/server/en/v1.0.0/http-client-server-api.html#get--api-v1-statuses?transaction_id=transaction_id)
+
+
+Changes:
+
+- All notion of `_links` was removed. In case of querying the status of a
+  transaction already included in a block marked `VALID`, no `_links` object is
+  provided anymore. The response object now only contains a single key value
+  pair named `status`
+- The query parameter `tx_id` was renamed to `transaction_id`, e.g. `GET
+  /api/v1/statuses?transaction_id=`
```

Comment:
I was consistently (throughout the document) lazy, but I can add that if you want